command.go

```diff
@@ -875,6 +875,13 @@ func (c *Command) ValidateArgs(args []string) error {
 	if c.Args == nil {
 		return nil
 	}
+	if len(c.ValidArgs) > 0 {
```

Comment:
Just to sum up, @umarcor , please replace this logic with an un-exported validation function in args.go.  If anything it keeps the method cleaner.  The logic of this function should be that of the `OnlyValidArgs` function you deprecated.

Reply:
How should it be named? `validateValidArgs`?

Reply:
Meh that sounds a bit repetitive. `validateArgs` is fine with me.